Use correct icons in beatmap details

danielkariv opened the pull request:
In relation to #13968. 
Replacing incorrect icons in beatmap details panel to correct ones from BeatmapStatisticsIcon class.

Old:
![image](https://user-images.githubusercontent.com/38776931/129493071-e460428f-ad99-41ae-b640-8155a56156f7.png)

New:
![image](https://user-images.githubusercontent.com/38776931/129493111-2f927853-26e0-45b7-9a53-e0790d9c6c94.png)

danielkariv committed:
Correct icons in beatmap details

In relation to #13968. Replacing incorrect icons in beatmap details panel to correct ones from BeatmapStatisticsIcon class.

frenzibyte commented:
Missing the rings around the icon, see [osu!web](https://osu.ppy.sh/beatmapsets/1271180#osu/2727386):
![CleanShot 2021-08-16 at 00 20 30](https://user-images.githubusercontent.com/22781491/129493268-df5eaada-e366-4618-a18c-3983d1a17dd4.png)

frenzibyte reviewed (dismissed):
As per above. Also inconsistent code style, starting bracket (`{`) should be in its own line, see surrounding code.

danielkariv committed:
Updating beatmap details icons

changes the sizing and add yellow circle so the UI will fit more with osu-web style.

danielkariv commented:
Okay, fixed the styling of the icons.
<img width="321" alt="image" src="https://user-images.githubusercontent.com/38776931/129493697-d0acf0f5-b4ed-4bfc-96fb-6f47fd6f3f0e.png">

smoogipoo reviewed (approved):
lgtm, thanks!
